prep-javascript-loops/main.js

```diff
@@ -0,0 +1,78 @@
+// create your loops here.
+function whileLoop1() {
+  const array = [];
+  let i = 0;
+  while (i <= 9) {
+    array.push(i);
+    i++;
+  }
+  return array;
+}
+
+console.log('whileLoop1 output', whileLoop1());
+
+function whileLoop2() {
+  const array = [];
+  let i = 0;
+  while (i <= 18) {
+    array.push(i);
+    i++;
+    i++;
+  }
+  return array;
+}
+
+console.log('whileLoop2 output', whileLoop2());
+
+function forLoop1() {
+  const array = [];
+  for (let i = 0; i <= 9; i++) {
+    array.push(i);
+  }
+  return array;
+}
+
+console.log('forLoop1 output', forLoop1());
+
+function forLoop2() {
+  const array = [];
+  for (let i = 100; i >= 0; i--) {
+    const message = 'Time till explosion ' + i + '!';
```

Comment:
this works for creating an array of strings!

This function specifically just need to log the messages to the console. Watch those details!